llama_stack/distribution/build_container.sh

```diff
@@ -166,17 +172,17 @@ EOF
   # and update the configuration to reference the new container path
   python_cmd=$(get_python_cmd)
   external_providers_dir=$($python_cmd -c "import yaml; config = yaml.safe_load(open('$run_config')); print(config.get('external_providers_dir') or '')")
-  if [ -n "$external_providers_dir" ]; then
+  if [ -n "$external_providers_dir" ] && [ -d "$external_providers_dir" ]; then
```

Comment:
This won't work with the new default `~/.llama/providers.d`, we need to force tilde expansion, otherwise `-d` will always be false.

Reply:
resolved in most recent commit